Content.Client/GameObjects/Components/Atmos/GasCanisterWindow.cs

```diff
@@ -0,0 +1,160 @@
+                                                new ReleasePressureButton() {PressureChange = -1},
+                                                new ReleasePressureButton() {PressureChange = -0.1f},
+                                                new ReleasePressureButton() {PressureChange = 0.1f},
+                                                new ReleasePressureButton() {PressureChange = 1},
+                                                new ReleasePressureButton() {PressureChange = 10},
+                                                new ReleasePressureButton() {PressureChange = 50}
+                                            }
+                                        })
+                                    }
+                                },
+                                new HBoxContainer()
+                                {
+                                    Children =
+                                    {
+                                        new Label { Text = Loc.GetString("Valve") },
+                                        (ToggleValve = new CheckButton() { Text = Loc.GetString("Open") })
+                                    }
+                                }
+                            },
+                        }
+                }
+            });
+
+            // Create the release pressure buttons list
+            ReleasePressureButtons = new List<ReleasePressureButton>();
+            foreach (var control in releasePressureButtons.Children.ToList())
+            {
+                var btn = (ReleasePressureButton) control;
+                ReleasePressureButtons.Add(btn);
+            }
+        }
+
+
+        /// <summary>
+        /// Update the UI based on <see cref="GasCanisterBoundUserInterfaceState"/>
+        /// </summary>
+        /// <param name="state">The state the UI should reflect</param>
+        public void UpdateState(GasCanisterBoundUserInterfaceState state)
+        {
+            _pressure.Text = state.Volume + "kPa";
+            _releasePressure.Text = state.ReleasePressure + "kPa";
+
+            // Update the canister label
+            OldLabel = LabelInput.Text;
+            LabelInput.Text = state.Label;
+            Title = state.Label;
+
+            // Reset the editable label
+            LabelInput.Editable = false;
+            EditLabelBtn.Text = EditLabelBtnStateEdit;
+
+            ToggleValve.Pressed = state.ValveOpened;
+        }
+    }
+
+
+    /// <summary>
+    /// Special button class which stores a numerical value and has it as a label
+    /// </summary>
+    public class ReleasePressureButton : Button
+    {
+        public float PressureChange
+        {
+            get { return _pressureChange; }
+            set
+            {
+                string prefix = value.Equals(Math.Abs(value)) ? "+" : "-";
+                Text = prefix + Math.Abs(value);
+                _pressureChange = value;
```

Comment:
This could use a few improvements like
- Using Math.Sign
- Only handling positive numbers (negative numbers and zero can be converted to string directly without "prefix")